build/scoring_test.go

```diff
@@ -572,7 +624,10 @@ func checkScoring(t *testing.T, c scoreCase, parserType ctags.CTagsParserType) {
 		}
 		defer ss.Close()
 
-		srs, err := ss.Search(context.Background(), c.query, &zoekt.SearchOptions{DebugScore: true})
+		srs, err := ss.Search(context.Background(), c.query, &zoekt.SearchOptions{
+			UseKeywordScoring: keywordScoring,
+			ChunkMatches:      true,
```

Comment:
I switched to `ChunkMatches: true` so the test is more representative of how we run in Sourcegraph.